chatterbot/conversation.py

```diff
@@ -216,7 +216,7 @@ def get_sentiment(name):
         conversation.
         """
         sentiment = []
-        for statement in self:
+        for statement in name:
```

Comment:
This was set to `self` before because the `Conversation` object [defines an `__iter__` method](https://github.com/PaarthBatra/ChatterBot/blob/2de82a504ecdeae2790ba7226a2f86dbfff971dc/chatterbot/conversation.py#L90-L95) that yields each statement. This basically allowed you to get each statement in the conversation in a loop.

`name` is used to only get statements for a specific person in the conversation, and iterating over it will give you each individual character.